<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1553</width>
    <height>758</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="documentMode">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,99">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QPushButton" name="help_info_button">
        <property name="text">
         <string>‚ùìÂú∫ÊôØËØ¥Êòé</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="demo_button">
        <property name="text">
         <string>üì∞ÊºîÁ§∫</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="sizeIncrement">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>ÂçïË°®ÊãÜÂàÜÂú∫ÊôØ</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_8">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="release_info_button">
        <property name="text">
         <string>üí°Âú∫ÊôØÊõ¥Êñ∞‰ø°ÊÅØ</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="reset_button">
        <property name="text">
         <string>ÈáçÁΩÆÊâÄÊúâ</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QPushButton" name="add_table_button">
             <property name="text">
              <string>‚ûï</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Á¨¨‰∏ÄÊ≠• ÔºöÊãñÊãΩÊàñÁÇπÂáª‰∏ä‰º†Excel</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="step1_help_info_button">
             <property name="text">
              <string>‚ùìÁ¨¨‰∏ÄÊ≠•ÁöÑÂ∏ÆÂä©‰ø°ÊÅØ</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableWidget" name="tables_table">
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAsNeeded</enum>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAsNeeded</enum>
           </property>
           <property name="autoScroll">
            <bool>true</bool>
           </property>
           <property name="horizontalScrollMode">
            <enum>QAbstractItemView::ScrollPerItem</enum>
           </property>
           <property name="columnCount">
            <number>5</number>
           </property>
           <attribute name="verticalHeaderVisible">
            <bool>true</bool>
           </attribute>
           <column>
            <property name="text">
             <string>Ë°®Âêç</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>ÈÄâ‰∏≠Â∑•‰ΩúË°®</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Ê†áÈ¢òÊâÄÂú®Ë°å</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Êìç‰ΩúÊåâÈíÆ</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>__Ë°®Ë∑ØÂæÑ</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_3">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="add_split_cols_button">
             <property name="text">
              <string>‚ûï</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Á¨¨‰∫åÊ≠• ÔºöÊ∑ªÂä†ÊãÜÂàÜÂàó</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="step2_help_info_button">
             <property name="text">
              <string>‚ùìÁ¨¨‰∫åÊ≠•ÁöÑÂ∏ÆÂä©‰ø°ÊÅØ</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableWidget" name="split_cols_table">
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAsNeeded</enum>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAsNeeded</enum>
           </property>
           <property name="autoScroll">
            <bool>true</bool>
           </property>
           <property name="horizontalScrollMode">
            <enum>QAbstractItemView::ScrollPerItem</enum>
           </property>
           <property name="columnCount">
            <number>2</number>
           </property>
           <attribute name="verticalHeaderVisible">
            <bool>true</bool>
           </attribute>
           <column>
            <property name="text">
             <string>ÊãÜÂàÜÂàó</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Êìç‰ΩúÊåâÈíÆ</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="run_button">
            <property name="text">
             <string>‚ñ∂</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Á¨¨‰∏âÊ≠• ÔºöÊâßË°å‰∏é‰∏ãËΩΩ</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="step3_help_info_button">
            <property name="text">
             <string>‚ùìÁ¨¨‰∏âÊ≠•ÁöÑÂ∏ÆÂä©‰ø°ÊÅØ</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>ÁªìÊûúÊñá‰ª∂È¢ÑËßà</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="result_detail_text">
            <property name="text">
             <string>üö´ÊâßË°åÊÄªËÆ°ËÄóÊó∂Ôºö--Áßí</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_9">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="download_result_button">
            <property name="text">
             <string>üìÑ‰∏ãËΩΩÁªìÊûú</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="result_table">
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAsNeeded</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAsNeeded</enum>
          </property>
          <property name="autoScroll">
           <bool>true</bool>
          </property>
          <property name="horizontalScrollMode">
           <enum>QAbstractItemView::ScrollPerItem</enum>
          </property>
          <property name="columnCount">
           <number>2</number>
          </property>
          <attribute name="verticalHeaderVisible">
           <bool>true</bool>
          </attribute>
          <column>
           <property name="text">
            <string>ÊãÜÂàÜÊñá‰ª∂/sheet</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Ë°åÊï∞</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
